container_src/main.go

```diff
@@ -13,22 +15,37 @@ func runMapi(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	workspace := r.FormValue("workspace")
-	project := r.FormValue("project")
-	target := r.FormValue("target")
-	api_url := r.FormValue("api_url")
-	api_spec := r.FormValue("api_spec")
-	duration := r.FormValue("duration")
+	// fetch data and strip whitespace
+	workspace := strings.TrimSpace(r.FormValue("workspace"))
+	project := strings.TrimSpace(r.FormValue("project"))
+	target := strings.TrimSpace(r.FormValue("target"))
+	api_url := strings.TrimSpace(r.FormValue("api_url"))
+	api_spec := strings.TrimSpace(r.FormValue("api_spec"))
+	duration := strings.TrimSpace(r.FormValue("duration"))
 
 	if workspace == "" || project == "" || target == "" || api_url == "" || api_spec == "" || duration == "" {
 		http.Error(w, "All fields are required!", http.StatusBadRequest)
 		return
 	}
 
-	cmd := exec.Command("mapi", "run", fmt.Sprintf("%s/%s/%s %s %s --url %s", workspace, project, target, duration, api_spec, api_url))
+	// Need to run /usr/local/bin/mapi run workspace/project/target duration api_spec --url api_url
+	cmd := exec.Command("/usr/local/bin/mapi", "run", fmt.Sprintf("%s/%s/%s", workspace, project, target), duration, api_spec, "--url", api_url)
+
+	// join full command into a single variable
+	fullCommand := fmt.Sprintf("mapi run %s/%s/%s %s %s --url %s", workspace, project, target, duration, api_spec, api_url)
+
+	// // Get the string value of the environment variable MAYHEM_TOKEN
+	// // if it exists otherwise use an empty string
+	// Note: this is all probably unnecessary, but helped with debugging
+	mayhemToken := ""
```

Comment:
at some point I would like to use secrets (https://developers.cloudflare.com/workers/configuration/secrets/) for this, but for testing this is fine